Makefile

```diff
@@ -57,7 +57,7 @@ test: generate-mocks unit
 gendeepcopy:
 	go build -o $$GOPATH/bin/deepcopy-gen sigs.k8s.io/cluster-api-provider-aws/vendor/k8s.io/code-generator/cmd/deepcopy-gen
 	deepcopy-gen \
```

Comment:
thanks! we should actually rely on go generate for this now https://github.com/openshift/cluster-api-provider-aws/blob/master/pkg/apis/apis.go#L18 cc @spangenberg

Reply:
@enxebre thx, fixed.